src/cascadia/WindowsTerminal/AppHost.cpp

```diff
@@ -16,14 +16,13 @@ using namespace ::Microsoft::Console::Types;
 
 AppHost::AppHost() noexcept :
     _app{},
-    _logic{},
     _window{ nullptr }
 {

```

Comment:
I might actually cache Logic in this class: otherwise you’re getting a cross-dll WinRT projection call for every single `Logic()`. Seem fair?

Reply:
you're killin' me bud.

Reply:
k done.